compiler/AST/iterator.cpp

```diff
@@ -112,43 +112,81 @@ isSingleLoopIterator(FnSymbol* fn, Vec<BaseAST*>& asts) {
     return NULL;
   BlockStmt* singleFor = NULL;
   CallExpr* singleYield = NULL;
-  forv_Vec(BaseAST, ast, asts) {
+  forv_Vec(BaseAST, ast, asts)
+  {
```

Comment:
Can you revert the brace changes in this function? It's confusing to read with both styles in the same function. I have mods to this function in my vectorization patch so I'd strongly prefer reverting to same line brace rather than updating the whole function.